File: scripts/simple.py
def uftrace_begin(ctx):
    print("program begins...")

def uftrace_entry(ctx):
    func = ctx["name"]
    print("entry : " + func + "()")

def uftrace_exit(ctx):
    func = ctx["name"]
    print("exit  : " + func + "()")

def uftrace_end():
    print("program is finished")

File: scripts/simple.lua
function uftrace_begin(ctx)
    print('program begins...')
end

function uftrace_entry(ctx)
    func = ctx['name']
    print('entry : ' .. func .. '()')
end

function uftrace_exit(ctx)
    func = ctx['name']
    print('exit  : ' .. func .. '()')
end

function uftrace_end()
    print('program is finished')
end
